Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,184 @@
+          pradeo.detection.value: '{{json_event.message.content.deviceSystemStatusRecord.value}}'
+      - filter: '{{json_event.message.type == "DeviceNetworkStatusRecordUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceNetworkStatusRecord.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceNetworkStatusRecord.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceNetworkStatusRecord.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceNetworkStatusRecord.device.email}}'
+          pradeo.device.imei: '{{json_event.message.content.deviceNetworkStatusRecord.device.imei}}'
+          pradeo.device.byod: '{{json_event.message.content.deviceNetworkStatusRecord.device.byod}}'
+          pradeo.device.declaredOperatingSystem: '{{json_event.message.content.deviceNetworkStatusRecord.device.declaredOperatingSystem}}'
+          pradeo.device.declaredOperatingSystemVersion: '{{json_event.message.content.deviceNetworkStatusRecord.device.declaredOperatingSystemVersion}}'
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: '{{json_event.message.content.deviceNetworkStatusRecord.device.declaredOperatingSystemSecurityPatchDate}}'
+          pradeo.device.declaredModel: '{{json_event.message.content.deviceNetworkStatusRecord.device.declaredModel}}'
+          pradeo.device.lastConnection: '{{json_event.message.content.deviceNetworkStatusRecord.device.enrollmentStatus.lastConnection}}'
+          pradeo.device.coupled: '{{json_event.message.content.deviceNetworkStatusRecord.device.enrollmentStatus.coupled}}'
+          pradeo.device.mdmId: '{{json_event.message.content.deviceNetworkStatusRecord.device.emmDeviceInfo.externalId}}'
+          pradeo.device.emm: '{{json_event.message.content.deviceNetworkStatusRecord.device.emmDeviceInfo.emm}}'
+          pradeo.detection.status: '{{json_event.message.content.deviceNetworkStatusRecord.deviceNetworkStatus.name}}'
+          pradeo.detection.value: '{{json_event.message.content.deviceNetworkStatusRecord.value}}'
+      - filter: '{{json_event.message.type == "DeviceStatusHistoryUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceId}}'
+          pradeo.detection.status: '{{json_event.message.content.event.kind}}'
+          pradeo.detection.value: '{{json_event.message.content.event.value}}'
+  correlation:
+    actions:
+      - filter: '{{json_event.message.type == "DeviceCorrelationUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceCorrelation.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceCorrelation.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceCorrelation.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceCorrelation.device.email}}'
```

[`user.email`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email) is more suitable.
```suggestion
          user.email: '{{json_event.message.content.deviceCorrelation.device.email}}'
```